src/FilamentFullCalendarServiceProvider.php

```diff
@@ -2,21 +2,24 @@
 
 namespace Saade\FilamentFullCalendar;
 
-use Filament\PluginServiceProvider;
+use Filament\Support\Facades\FilamentAsset;
+use Filament\Support\Assets\Css;
+use Filament\Support\Assets\Js;
 use Saade\FilamentFullCalendar\Commands\UpgradeFilamentFullCalendarCommand;
 use Spatie\LaravelPackageTools\Package;
+use Spatie\LaravelPackageTools\PackageServiceProvider;
 
-class FilamentFullCalendarServiceProvider extends PluginServiceProvider
+class FilamentFullCalendarServiceProvider extends PackageServiceProvider
 {
     public static string $name = 'filament-fullcalendar';
 
-    protected array $beforeCoreScripts = [
-        'filament-fullcalendar-scripts' => __DIR__ . '/../dist/filament-fullcalendar.js',
-    ];
-
-    protected array $styles = [
-        'filament-fullcalendar-styles' => __DIR__ . '/../dist/filament-fullcalendar.css',
-    ];
+    public function packageRegistered(): void
```

Comment:
```suggestion
    public function packageBooted(): void
```

Asset registration should go into the packageBooted method

See:

https://filamentphp.com/docs/3.x/support/plugins/build-a-panel-plugin

https://filamentphp.com/docs/3.x/support/plugins/build-a-standalone-plugin